cookieninja/vcs.py

```diff
@@ -41,6 +41,8 @@ def identify_repo(repo_url):
     else:
         if 'git' in repo_url:
             return 'git', repo_url
+        elif 'codecommit' in repo_url:
+            return 'git-remote-codecommit', repo_url
```

Comment:
Why the name is `git-remote-codecommit` and not `git-codecommit`?
Or only `codecommit`

Reply:
I guess that this is the desired name for AWS code commit repos...